tests/functional/test_install.py

```diff
@@ -634,7 +634,7 @@ def test_install_global_option(script):
     result = script.pip(
         'install', '--global-option=--version', "INITools==0.1",
         expect_stderr=True)
-    assert 'INITools==0.1\n' in result.stdout
+    assert not result.files_created
```

Comment:
I would keep the original test as well, just to ensure we aren't changing behavior.

Reply:
amended that